docs/decex/staking.md

```diff
@@ -24,51 +24,83 @@ NPoS allows for virtually all SSC holders to participate, thus maintaining high
+    4. The $\text{shares}$ of this nominator are added to $\text{total\_shares}$ of the operator's pool.
 
-The nomination pools in Subspace are "lazy": any rewards earned by the operator are assigned to the pool and are not deposited to the nominators wallet unless they ask for a withdrawal. Unless withdrawn, the rewards are "auto-staked" - they count towards the total stake of the pool, increasing its chance of being elected to produce bundles.
+The nomination pools in Subspace are "lazy": any fees earned by the operator are assigned to the pool and are not deposited to the nominators wallet unless they ask for a withdrawal. Unless withdrawn, the fees are "auto-staked" - they count towards the total stake of the pool, increasing its chance of being elected to produce bundles.
 
-When the nominator decides to withdraw their stake or rewards, they submit a withdraw extrinsic. The withdraw extrinsic is processed at the end of the epoch and the stake is removed from the operator's pool and the domain's total stake. The nominator is then entitled to the rewards based on the stake shares and the amount of time they have staked.
+When the nominator decides to withdraw their stake or fees, they submit a withdraw extrinsic. The withdraw extrinsic is processed at the end of the epoch and the stake is removed from the operator's pool and the domain's total stake. The nominator is then entitled to the fees percentage based on the stake shares and the amount of time they have staked.
 
-Operators can also withdraw their stake and rewards at any time by submitting a withdraw extrinsic. If their withdrawal leaves their stake below the domain's minimum requirements, the operator will be removed from the domain and their stake and the stakes of all nominators will be returned to their wallets.
+Operators can also withdraw their stake and fees at any time by submitting a `withdraw_stake` extrinsic. Operators who wish to withdraw all of their stake and earned fees have to submit a deregistration extrinsic, as it is forbidden to withdraw below the domain's minimum stake requirements. The deregistered operator will be removed from the domain and their stake and the stakes of all nominators will be returned to their accounts.
 
 <div align="center">
     <img src="/img/Nomination_Pool-light.svg#gh-light-mode-only" alt="Nomination_Pool" />
     <img src="/img/Nomination_Pool-dark.svg#gh-dark-mode-only" alt="Nomination_Pool" />
 </div>
 
+### Unlocking Funds
+
+Withdrawals have a lock period of 2 days (currently 28 800 blocks, ~ 48 hours). After the locking period, the withdrawn amount can be unlocked in the user's account with the `unlock_funds` extrinsic. All withdrawals requested in the same stake epoch are aggregated together and the total amount is unlocked at once. This locking period is necessary to ensure that the domain block executing the withdrawal is confirmed and not challenged by a fraud proof in order to increase the economic stability of domains. 
```

Comment:
I would like to clarify my understanding here. 
Previously, in order to unlock tokens operators:
1 -  Can either withdraw tokens using `withdrawStake(operatorId, withdraw)`
2 - Or can withdraw stake by de-registering `deregisterOperator(operatorID)`
In both cases their funds were automatically unlocked after the locking period. 
Would that change at all now (other than the locking time changes)? It sounds like the funds won't be automatically unlocked, but users have to use another extinsics call to `unlock_funds ` for the tokens to be released. Maybe I'm misinterpreting something.

Reply:
Yes. After 1 or 2, the user has to wait and then submit `unlock_funds` to actually be able to use the tokens. `withdrawStake` now basically only "unstakes" from the pool, but doesn't actually compute the individual amount or mint/transfer. This change is worse UX but a huge optimization to domain that lets us avoid iterating through all hundreds (or thousands) of nominators every time.
Currently, the `unlock_funds` has to be submitted manually, but we could have it automated by the staking interface or wallet so that the funds that are ready to be unlocked show like a reminder icon for the user or even submitted automatically as `future` transaction. There's room for UX improvement for sure.

Reply:
Makes sense, thank you!
I'm adding the step to the guide as well, at least until we automate it.